ocfweb/docs/docs/services/account/content-removal.md

```diff
@@ -0,0 +1,54 @@
+[[!meta title="Content Removal Policy"]]
+
+
+## Introduction
+
+Most of the content on [ocf.berkeley.edu](https://ocf.berkeley.edu)
+is created and owned by OCF users, not the OCF itself.
+
+The OCF maintains a strict policy not to remove or modify data
+hosted by members unless legally compelled to do so, or if such
+data is in violation of existing OCF [account policies][policy].
```

Comment:
we shouldn't link like this, we have the ability to do links from markdown to our own site via route name:

```markdown
existing OCF [[account policies|doc services/account/account-policies]]
```

there's a few benefits:

* guaranteed no broken links (the test would fail)
* links to the right host (e.g. if this is a dev instance, it still links to the dev host)